### DISCUSSION TITLE
Bar Chart Data Zoom

### DISCUSSION THREAD

**Discussion by caiotaulois** [apache/superset#20120] [Q&A / Help]
Hello!!

Is there a way to add "Data Zoom" to a "Bar Chart"? "Time-series Bar Chart v2" has this feature:

Customize:
![image](https://user-images.githubusercontent.com/12478956/169156733-7ea78eb7-55c8-4fed-bb7d-a5f010028932.png)

Chart:
![image](https://user-images.githubusercontent.com/12478956/169156554-7b57bee7-6175-4d41-9763-2cfc6cfbc57b.png)

I need a bar chart with so many data that's it's impossible to see anything without this feature:

![image](https://user-images.githubusercontent.com/12478956/169156880-38d512b7-318b-49db-b796-870eb7a0be66.png)

Thank you all in advance.

**Comment by kamalkeshavani-aiinside** [comment]
You can enable `GENERIC_CHART_AXES` feature flag, which will allow to use non-timestamp values in X-axis like Bar Chart. I think since Superset is moving to Echarts, new features will not be added to legacy charts rather they will be removed.

**Reply by caiotaulois** [reply]
Thanks! I'll try it.
